libs/MVS/SceneTexture.cpp

```diff
@@ -509,11 +519,36 @@ bool MeshTexture::ListCameraFaces(FaceDataViewArr& facesDatas, float fOutlierThr
 		faceMap.create(imageData.height, imageData.width);
 		depthMap.create(imageData.height, imageData.width);
 		RasterMesh rasterer(vertices, imageData.camera, depthMap, faceMap);
+		//Creating mask for the image border
+		cv::cvtColor(imageData.image, rasterer.mask, cv::COLOR_BGR2GRAY);
+		rasterer.mask -= 1; // This line gets rid of white spots in the middle of the image
+
+		cv::floodFill(rasterer.mask, cv::Point(0, 0), 255);
+		cv::floodFill(rasterer.mask, cv::Point(0, rasterer.mask.rows / 2), 255);
+		cv::floodFill(rasterer.mask, cv::Point(0, rasterer.mask.rows - 1), 255);
+		cv::floodFill(rasterer.mask, cv::Point(rasterer.mask.cols / 2, rasterer.mask.rows - 1), 255);
+		cv::floodFill(rasterer.mask, cv::Point(rasterer.mask.cols - 1, rasterer.mask.rows - 1), 255);
+		cv::floodFill(rasterer.mask, cv::Point(rasterer.mask.cols - 1, rasterer.mask.rows / 2), 255);
+		cv::floodFill(rasterer.mask, cv::Point(rasterer.mask.cols - 1, 0), 255);
+		cv::floodFill(rasterer.mask, cv::Point(rasterer.mask.cols / 2, 0), 255);
+
+		cv::Mat kernel = cv::getStructuringElement(cv::MORPH_ELLIPSE, cv::Size(3, 3));
+		cv::erode(rasterer.mask, rasterer.mask, kernel);
+
+		cv::threshold(rasterer.mask, rasterer.mask, 254, 255, cv::THRESH_BINARY);
+
+
+
+		if (VERBOSITY_LEVEL > 2)
+			cv::imwrite(std::to_string(idx) + ".jpg", rasterer.mask);
```

Comment:
pls add some name too, like invalidmaskXXXX.png
use String::FormatString(), see code for ex